lib/WebService/Async/Onfido.pm

```diff
@@ -830,6 +835,73 @@ sub report_list {
     return $src;
 }
 
+=head2 countries_list
+
+Returns a hashref containing 2-letter country codes as keys and supporting status
+as their value.
+
+=cut
+
+sub countries_list {
+    my ($self) = @_;
+
+    $self->ua->GET(SUPPORTED_COUNTRIES_URL)->then(sub {
+        try {
+            my ($res) = @_;
+            my $onfido_countries = decode_json_utf8($res->content);
+
+            my $countries_list = {};
+            $countries_list->{uc(country_code2code($_->{alpha3}, 'alpha-3', 'alpha-2'))} = $_->{supported_identity_report} + 0 for @$onfido_countries;
```

Comment:
`map` would be slightly neater, although the end result is the same:

```suggestion
            my %countries_list = map { uc(country_code2code($_->{alpha3}, 'alpha-3', 'alpha-2')) => $_->{supported_identity_report} + 0 } @$onfido_countries;
```

Reply:
Great, thanks. 👍